src/support/json.c

```diff
@@ -170,8 +171,12 @@ __json_unpack_put(WT_SESSION_IMPL *session, void *voidpv, u_char *buf, size_t bu
                 }
                 s += n;
             }
-        if (bufsz > 0)
+        if (bufsz > 0) {
+            /* Expect just enough remaining buffer for closing quotes and terminator. */
```

Comment:
Just to understand why is `bufsz == 2` an assertion here?

Reply:
From my understanding bufsz is calculated in advance, and decremented when each character is added to the buffer, so if the calculation is correct the expected _remaining_ buffer size should be 2 (in theory 1 remaining after the `"` and 0 remaining after the `\0`).

Reply:
That is a great explanation, could you add that into the comment? For example:
```
bufsz is calculated in advance, and decremented when each character is added to the buffer, therefore we should only expect the remaining buffer to have closing quotes and a NULL terminator
```

Reply:
Done